README.rst

```diff
@@ -44,8 +51,12 @@ Multiformats_ and related packages:
 Building
 ================================================================================
 
-We maintain both a cabal_ and a stack_ build. Please see the package READMEs for
-additional instructions, specifically test setups.
+The canonical build for this project is using cabal_ (>= 2.4, "new-build). For
+convencience, we also provide a ``stack.yaml``. stack_ >= 1.9.1 is required to
+meet the version requirements for ``Cabal``.
```

Comment:
Should we also provide a stock `.freeze` file (produced with `cabal new-freeze`) for conveniency?

Reply:
Not sure -- this is essentially the same as a stackage snapshot. One reason to not use stack for a (mostly) library project is to ensure (to some extent) that it builds against whatever the solver chooses.